### PR TITLE
fix: remove an irrelevant section

### LINKED ISSUE #14273: Issue with "Mastering Wrapping of Flex Items": Contains information irrelevant for the topic
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Mastering_Wrapping_of_Flex_Items

#### What information was incorrect, unhelpful, or incomplete?
The last section is not related to this article

#### Specific section or headline?
[The difference between visibility: hidden and display: none](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Mastering_Wrapping_of_Flex_Items#the_difference_between_visibility_hidden_and_display_none)

#### What did you expect to see?
Only information related to the topic: Mastering Wrapping of Flex Items

#### Did you test this? If so, how?
The information is correct, but irrelevant for this topic

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Mastering_Wrapping_of_Flex_Items
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items/index.md
* Last commit: https://github.com/mdn/content/commit/426485595b1be50e2af91764fbfd94c04c3c36a5
* Document last modified: 2022-02-18T11:38:26.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fixes https://github.com/mdn/content/issues/14273

### CODE REVIEW

**Comment by bsmth** [comment]
I believe this is important in flex layouts where using `visibility: hidden` v `display: none` has implications that might not be so obvious at first: https://codepen.io/bsmth/pen/eYbpmeB

**Comment by OnkarRuikar** [comment]
> I believe this is important in flex layouts

I know. But looking at who re-labeled the issue instead of closing it I thought it has to be addressed.\
As extra info never hurts, we can close the issue as won't fix.

**Comment by OnkarRuikar** [comment]
>  there's probably room to expand on details or add an example if it's not clear that it's relevant for readers, what do you reckon?

I didn't realize it was a sub section under the collapse section. I've rephrased it to bring closer to the context and not to look like comparison between unrelated properties.

File: files/en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items/index.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ I have created this behavior in the next live example. You can see how the stret
...
 
-### The difference between `visibility: hidden` and `display: none`
+### Using `visibility: hidden` and `display: none`
 
-When you set an item to `display: none` in order to hide it, the item is removed from the formatting structure of the page. What this means in practice is that counters ignore it, and things like transitions do not run. Using `visibility: hidden` keeps the box in the formatting structure which is useful in that it still behaves as if it were part of the layout even though the user can't see it.
+In previous live example, try using `visibility: hidden` or `display: none` instead of `visiblity: collapse`. Using `visibility: hidden`, the item is made invisible but the box is kept in the formatting structure, so it still behaves as if it were part of the layout.
+When you use `display: none`, the item is completely removed from the formatting structure. Not only it is invisible but the strut is removed as well. What this means in practice is that counters ignore it, and things like transitions do not run.
```
Comment (estelle) [inline files/en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items/index.md:92]:
```suggestion
When you use `display: none`, the item is completely removed from the formatting structure. Not only is it invisible but the structure is removed as well. This means counters ignore it and things like transitions do not run.
```

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Comment by bsmth** [comment]
I'm going to merge this as there's already an approval and the suggested changes are applied. Thanks both 👍🏻

**Pull request merged**